src/module.c

```diff
@@ -14857,16 +14857,16 @@ void moduleRegisterCoreAPI(void) {
     REGISTER_API(RdbLoad);
     REGISTER_API(RdbSave);
     REGISTER_API(GetInternalSecret);
-    REGISTER_API(GetConfigIterator);
-    REGISTER_API(ReleaseConfigIterator);
+    REGISTER_API(ConfigGetIterator);
+    REGISTER_API(ConfigReleaseIterator);
```

Comment:
again aiming for common prefix..
```suggestion
    REGISTER_API(ConfigIteratorCreate);
    REGISTER_API(ConfigIteratorRelease);
```

Reply:
ok makes sense